stext-device: do not add a combining mark when there is no current line

cur_line can be NULL here, and add_char_to_line() dereferences it. It happens
when a combining mark arrives before any line has been started.
---
diff --git a/source/fitz/stext-device.c b/source/fitz/stext-device.c
index 5ca5ca4..027a535 100644
--- a/source/fitz/stext-device.c
+++ b/source/fitz/stext-device.c
@@ -826,7 +826,7 @@ fz_add_stext_char_imp(fz_context *ctx, fz_stext_device *dev, fz_font *font, int
 	}
 
 	/* Don't move pen for marking non-spacing characters */
-	if (ucdn_get_general_category(c) == UCDN_GENERAL_CATEGORY_MN)
+	if (cur_line && ucdn_get_general_category(c) == UCDN_GENERAL_CATEGORY_MN)
 	{
 		add_char_to_line(ctx, page, cur_line, trm, font, size, c, (dev->flags & FZ_STEXT_ACCURATE_BBOXES) ? glyph : NON_ACCURATE_GLYPH, &dev->pen, &dev->pen, bidi, dev->color, 0, flags, dev->flags);
 		dev->lastbidi = bidi;
